addons/web/static/src/core/domain_selector/domain_selector.js

```diff
@@ -177,16 +193,47 @@ export class DomainSelector extends Component {
             options
         ).children[0];
 
+        this.includeArchived = false;
+        if (this.tree.value === "&") {
```

Comment:
is this necessary if showArchivedCheckbox is false?

Reply:
It is not indeed